controllers/nginx/pkg/config/config.go

```diff
@@ -307,20 +301,14 @@ func NewDefault() Configuration {
 	return cfg
 }
 
-// BuildLogFormatUpstream format the log_format upstream based on proxy_protocol
-func BuildLogFormatUpstream(useProxyProtocol bool, curLogFormatUpstream string) string {
-
-	// test if log_format comes from configmap
-	if curLogFormatUpstream != "" &&
-		curLogFormatUpstream != fmt.Sprintf(logFormatUpstream, "$proxy_protocol_addr") &&
-		curLogFormatUpstream != fmt.Sprintf(logFormatUpstream, "$remote_addr") {
-		return curLogFormatUpstream
-	}
-
-	if useProxyProtocol {
-		return fmt.Sprintf(logFormatUpstream, "$proxy_protocol_addr")
+// BuildLogFormatUpstream format the log_format upstream using
+// proxy_protocol_addr as remote client address if UseProxyProtocol
+// is enabled.
+func (cfg Configuration) BuildLogFormatUpstream() string {
```

Comment:
@gianrubio no need. At this point the configuration contains the default value or the defined in the configmap.
```
apiVersion: v1
data:
  log-format-upstream: $remote_user
kind: ConfigMap
metadata:
  name: nginx-template
  namespace: default
```
```console
I0304 21:53:19.723598       5 nginx.go:224] --- /tmp/a080909879	2017-03-04 21:53:19.000000000 +0000
+++ /tmp/b283781162	2017-03-04 21:53:19.000000000 +0000
@@ -56,7 +56,7 @@

     server_tokens on;

-    log_format upstreaminfo '$remote_addr - [$proxy_add_x_forwarded_for] - $remote_user [$time_local] "$request" $status $body_bytes_sent "$http_referer" "$http_user_agent" $request_length $request_time [$proxy_upstream_name] $upstream_addr $upstream_response_length $upstream_response_time $upstream_status"';
+    log_format upstreaminfo '$remote_addr - $remote_user';

     map $request_uri $loggable {
         default 1;
I0304 21:53:20.358319       5 controller.go:426] ingress backend successfully reloaded...
```